src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,84 @@
-export const Pagination = () => {};
+import cn from 'classnames';
+
+type Props = {
+  total: number,
+  perPage : number,
+  currentPage: number,
```

Comment:
The prop 'currentPage' should be optional with a default value of 1 as required by the task.